File: 백준/Bronze/9295. 주사위/주사위.js
const fs = require('fs')
const input = fs.readFileSync(process.platform === "linux" ? "/dev/stdin":"입력.txt")
  .toString().trim().split('\n')

function solution(data) {
  data.shift()
  const result = data.map((el, idx) => {
    const [a,b] = el.split(' ').map(Number)
    return `Case ${idx + 1}: ${a + b}`
  })

  console.log(result.join('\n'))
}

solution(input) 

File: 백준/Bronze/1233. 주사위/주사위.py
import sys
data_temp = sys.stdin if sys.platform == 'linux' else open('입력.txt', 'r')
input_data = data_temp.read()

def solution (data) :
  a,b,c = map(int, data.split())
  dic = {}
  
  for i in range(a) :
    for j in range(b) :
      for k in range(c) : 
        temp = i + j + k + 3
        dic[temp] = dic.get(temp, 0) + 1
    
  _max, result = 0, 0

  for i in range(3, a+b+c+1) :
    if dic[i] > _max : 
      _max = dic[i]
      result = i
    else : break
  
  print(result)
  
solution(input_data)

